enable/markers.py

```diff
@@ -323,7 +322,7 @@ class StarMarker(AbstractMarker):
     """ A marker that is a (filled) star.
     """
 
-    # How this marker is to be stroked. (Overrides AbstractMarker.)
+    #: How this marker is to be stroked. (Overrides AbstractMarker.)
     draw_mode = FILL_STROKE
     # Do not render anti-aliased. (Overrides AbstractMarker.)
     antialias = True
```

Comment:
this - and more such examples are ignored in this PR - because the comment strings say one thing (i.e. `antialias = False`) but the actual code says something else (i.e. `antialias = True`). This needs to be investigated further, separately.

Reply:
Update : These markers were added in https://github.com/enthought/enable/pull/222 - which flipped `antialias` back to `True` but the PR forgot to remove the outdated comments.